### ISSUE TITLE
Bug when specifing "multiple" to a complex command

### ISSUE THREAD

**Issue by yoav-steinberg** [redis/redis-io#251]
This pr: https://github.com/redis/redis-doc/pull/1645
Caused the `GET` subcommand to replicate each time the `BITFIELD_RO` page was refreshed. See attached image.

**Comment by yoav-steinberg** [comment]
I think the issue happens when you have a complex subcommand which has more than one argument but can appear multiple time. Relevant commands are `BITFIELD` and `BITFIELD_RO`. If there are others please add them to this ticket.
Once this is fixed we'll update `commands.json` accordingly.

**Comment by huangzhw** [comment]
I test in `help.h`, it is `key GET type offset [GET type offset ...]`.

**Comment by yoav-steinberg** [comment]
> I test in `help.h`, it is `key GET type offset [GET type offset ...]`.

Yes, that's the correct usage of the command but for some reason when specifying `"multiple":true` in `commands.json` and we refresh the command's page a few times we get the above issue.

**Comment by itamarhaber** [comment]
After multiple refreshes, I could not reproduce :/

**Comment by yoav-steinberg** [comment]
Recreates for me (chromium and firefox). Apply following patch, run the site and go to: http://localhost:9292/commands/bitfield_ro
```diff
diff --git a/commands.json b/commands.json
index 4f40282..4fb49e0 100644
--- a/commands.json
+++ b/commands.json
@@ -278,7 +278,8 @@
         "type": [
           "type",
           "integer"
-        ]
+        ],
+       "multiple": true
       }
     ],
     "since": "6.2.0",
```

**Comment by itamarhaber** [comment]
Ok, I see it, confirmed :)
